src/utils/constants.js

```diff
@@ -0,0 +1,10 @@
+export const clashingProperties = {
```

Comment:
Slight issue with the way these are defined. The object has been declared as a constant but the values are still mutable and can be overwritten by anywhere in the code effectively making them not constants.
Might be better to define them individually e.g
```
const START_TIME = 'startTime';
const END_TIME = 'endTime';
etc.
```
Also allows us to then import only the ones which we need.

Reply:
I see what you mean, though it would still be nice to group them in some way. Do you think using Object.freeze would be an ok alternative? Pro: it allows grouping, con: you still have to import a whole group, not an individual constant?

Reply:
updated :)